src/Symfony/Component/HttpFoundation/AcceptHeader.php

```diff
@@ -18,155 +20,8 @@
  * sorted by descending quality.
  *
  * @author Jean-François Simon <contact@jfsimon.fr>
+ * @deprecated use Symfony\Component\HttpFoundation\Header\Accept instead
```

Comment:
Please keep a line between the phpdoc tags and should be `@deprecated since 2.8, to be removed in 3.0. Use Symfony\Component\HttpFoundation\Header\Accept instead.`